frontend/src/metabase/collections/components/CollectionContent/CollectionContent.tsx

```diff
@@ -47,7 +47,7 @@ export function CollectionContent({
     getSetting(state, "uploads-enabled"),
   );
 
-  const canUploadToDb = useSelector(
+  const canCreateUploadInDb = useSelector(
```

Comment:
I'm curious, what does `canCreateUploadInDb` convey that `canUploadToDb` doesn't?

Reply:
This is my attempt to help prevent the confusion that created the bug in the first place: `database.can_upload = true` actually only means "we can create a new uploaded table", but instead you thought it meant "we can create to a new uploaded table and we can append to an existing table". Making this variable `canCreateUploadInDb` removes that confusion, I think.